docs/DeveloperGuide.md

```diff
@@ -2,12 +2,27 @@
 
 ## Acknowledgements
 
+* [AB-3 Developer Guide](https://se-education.org/addressbook-level3/DeveloperGuide.html)
+* [PlantUML Tutorial at se-edu/guides](https://se-education.org/guides/tutorials/plantUml.html)
+* [Our individual projects](AboutUs.md)
+
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Study Manager component
+API: `StudyManager.java`
+
+![](images/StudyManagerClass.png)
+
+The `StudyManager` component,
+1. Stores the academic schedule, i.e. all `Module` objects are contained in a `modulesList` object.
+2. The storage can be bound to `modulesList` list such that everytime a change is observed in the list it is saved on to a file.
+3. Does not depend on other components like `ExpenseTracker` and `ContactsManager`
+
+
 
 ## Product scope
 ### Target user profile
```

Comment:
Is the 'Product Scope' section wrongly positioned? I noticed that currently, the 'Main Menu' section is a subheader of 'Product Scope', and I presume that the former should be under the 'Design and Implementation' section.